common/views/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+import { classNames } from '../../../utils/classnames';
+import Space from '../styled/Space';
+
+const CheckboxLabel = styled.label.attrs({
+  className: classNames({
+    'flex flex--v-center': true,
+  }),
+})``;
+
+const CheckboxBox = styled.span.attrs({
+  className: classNames({
+    CheckboxBox: true,
+    'flex-inline flex--v-center flex--h-center relative': true,
```

Comment:
How do we feel about separate vs not separate fields?

I felt it was easier just to say 1 prop per class, and it might land up with analysis or find / replace etc easier, but I haven't got any evidence for that.

Reply:
Can you explain this a bit more? I don't follow.

Reply:
When do we use:
`{ 'flex-inline flex--v-center flex--h-center relative': true }`
vs
```js
{
    'flex-inline': true,
    'flex--v-center': true,
    'flex--h-center': true,
    'relative': true   
}
```

Reply:
Ah, I see. I've been comfortable grouping them into the largest number that could be dealt with by the one Boolean. But I'm easy.

Reply:
Ditto